src/storage/QueryBaseProcessor.inl

```
@@ -366,7 +366,7 @@ kvstore::ResultCode QueryBaseProcessor<REQ, RESP>::collectEdgeProps(
                 // TODO(heng): We could remove the lock with one filter one bucket.
                 std::lock_guard<std::mutex> lg(this->lock_);
                 auto& getters = expCtx_->getters();
-                getters.getEdgeProp = [&] (const std::string &prop) -> VariantType {
+                getters.getEdgeProp = [&] (const std::string &prop) -> OptVariantType {
                     auto res = RowReader::getPropByName(reader.get(), prop);
                     CHECK(ok(res));
```

Could we return an error to replace this ```CHECK```?